src/ProjectOrigin.VerifiableEventStore/Services/Repository/ITransactionRepository.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ProjectOrigin.VerifiableEventStore.Models;
+
+namespace ProjectOrigin.VerifiableEventStore.Services.Repository;
+
+public interface ITransactionRepository
+{
+    Task Store(StreamTransaction @event);
+
+    Task<ImmutableLog.V1.Block?> GetBlockFromTransactionHash(TransactionHash transactionHash);
+    Task<TransactionStatus> GetTransactionStatus(TransactionHash transactionHash);
```

Comment:
The naming of these two differ a bit. I suggest to remove the `FromTransactionHash` part of `GetBlockFromTransactionHash`

Reply:
Fixed